picard/ui/options/plugins.py

```diff
@@ -366,7 +366,7 @@ def open_plugins(self):
         )
         if files:
             for path in files:
-                self.tagger.pluginmanager.install_plugin(path)
+	            self.tagger.pluginmanager.install_plugin(path, action=PLUGIN_ACTION_INSTALL)
```

Comment:
Indentation is incorrect here

Reply:
Ah sorry, my sublime settings got reset. It ended up inserting a tab there instead of 4 spaces. Will fix.

Reply:
Done.